crates/cadenza-syntax/src/generated/snapshots/cadenza_syntax__generated__test_data__record_in_application__record_in_application_ast.snap

```diff
@@ -0,0 +1,9 @@
+---
+source: crates/cadenza-syntax/src/generated/test_data.rs
+expression: "foo 1 {\n  a = 1\n} 2\nbar\n"
+---
+[
+    [[foo, 1], [__record__, [=, a, 1]]],
+    2,
+    bar,
```

Comment:
Similar to the list one - this should continue the juxtaposition on the next line because the newline started with the expected closing punctuation. So we should only see two top-level expressions in this case. One where the final argument is `2`. And then `bar` should be in its own expression.